lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3329,7 +3329,7 @@ namespace {
 
       auto result = FuncDecl::create(
           Impl.SwiftContext, /*StaticLoc=*/SourceLoc(), StaticSpellingKind::None,
-          /*FuncLoc=*/SourceLoc(), name, /*NameLoc=*/SourceLoc(),
+          /*FuncLoc=*/SourceLoc(), importedName.Imported, /*NameLoc=*/SourceLoc(),
```

Comment:
With my recent PR that got merged yesterday: `importedName.getDeclName()`. Alternatively, there's still the implicit cast I kept around from before.